boards/xtensa/intel_adsp_cavs18/doc/generic_intel_adsp.rst

```diff
@@ -27,6 +27,27 @@ Set up the environment
      https://github.com/thesofproject/rimage.
 
 
+How Remote Service works
```

Comment:
Maybe I'm missing something, why do we have two separate docs, one for 15 and one for generic CAVS? Can we delete the 15 one or point it to the generic doc in 18?

Reply:
Thanks the suggestions! @laurenmurphyx64. I have another PR depending on this, so I will try to regular them after this PR.